packages/open-payments/src/client/index.ts

```diff
@@ -145,8 +192,8 @@ export const createUnauthenticatedClient = async (
 
 export interface CreateAuthenticatedClientArgs
   extends CreateUnauthenticatedClientArgs {
-  /** The private EdDSA-Ed25519 key with which requests will be signed */
-  privateKey: KeyLike
+  /** The path to the file containing the EdDSA-Ed25519 key with which requests will be signed */
+  privateKey: string | KeyLike
```

Comment:
a  bit redundant because KeyLike is just a type of `string | Buffer | KeyObject` but I like the explicitness